Harvest.Web/ClientApp/src/Projects/ProjectDetailContainer.tsx

```diff
@@ -73,7 +73,7 @@ export const ProjectDetailContainer = () => {
             <div className="col">
               <ShowFor
                 roles={["FieldManager", "Supervisor"]}
-                condition={project.status === "Requested"}
+                condition={project.status === "Requested" || project.status === "ChangeRequested"}
```

Comment:
@sprucely could you use the `ProjectStatus` enum typescript type here so we don't have to use the strings?

Reply:
ProjectStatus is a union of string literals. Are you asking me to change it to an enum?